### ISSUE TITLE
Binaries for Qt Creator 4.12.4

### ISSUE THREAD

**Issue by mbnoimi** [fpoussin/qtcreator-doxygen#37]
May you please release binaries for Qt Creator 4.12.4

I tried to build it by myself by I failed to build it correctly although I set a valid values to `IDE_BUILD_TREE` and `QTCREATOR_SOURCES`

**Comment by lamar-nyckmaia** [comment]
Yes...I need it too.
For both: 
 - Ubuntu 20 
 - OSX Catalina

**Comment by TomMalfrere** [comment]
Have you tried the Doxygen4.dll from the binaries section?
I'm using this on QtCreator 4.12.4 on Windows10...

**Comment by mbnoimi** [comment]
> Have you tried the Doxygen4.dll from the binaries section?
> I'm using this on QtCreator 4.12.4 on Windows10...

Sorry I didn't mention; I use Linux Mint 20

**Comment by francescmm** [comment]
Hi @lamar-nyckmaia @mbnoimi I've forked the repo and I've done the binaries myself. I hope they work:

https://github.com/francescmm/qtcreator-doxygen/releases/tag/v4.8.0

**Comment by nyckmaia** [comment]
great @francescmm !

**Comment by nyckmaia** [comment]
@francescmm the binary for Qt Creator 4.12.4 is not working on OSX Catalina.
Beside that, I'm not founding this plugin for Qt Creator 4.12.4 to run on Windows 10 x64.

**Comment by francescmm** [comment]
Hi @nyckmaia I totally forgot to update the Windows one, but I fix that! 😄 Could you try that version, please?

About OSX Catalina I don't know what could be happening. I did the plugin using the CI so I might have the wrong configuration there. Could you [open an issue about it in my fork](https://github.com/francescmm/qtcreator-doxygen/issues)? It would be helpful to have an extended description of the problem so I can try to fix it in the CI.

Thanks!

**Comment by nyckmaia** [comment]
Hi @francescmm, Today I downloaded and tested both versions: Windows 10 and OSX Catalina

On Windows 10 x64 I got this Qt Creator Load plugins error message:
```
C:\Users\VM\AppData\Local\QtProject\qtcreator\plugins\4.12.4\Doxygen4-x64.dll: Cannot load library 
C:\Users\VM\AppData\Local\QtProject\qtcreator\plugins\4.12.4\Doxygen4-x64.dll: Unknown error 0x000000c1.
```

On OSX Catalina, I got this Load Plugins error message:
![Error Plugin](https://user-images.githubusercontent.com/4555404/90172064-d5bd7a00-dd78-11ea-9ef7-5c99fd23a40f.jpeg)

**Comment by francescmm** [comment]
Hi @nyckmaia I've updated the plugins and generated the [QtCreator 4.12 x64 for Windows](https://github.com/francescmm/qtcreator-doxygen/releases/download/v4.8.0/libDoxygen_0.4.8-qtc4.12_x64.dll) that was the missing one.

About the OSX I'll do the investigation next week, tbh I don't have a proper OSX to test it, so it's just generated by the CI.

I hope the Win10 64 works 😄

**Comment by nyckmaia** [comment]
@francescmm thank you for your quick response!

Today I deleted the old Doxygen plugin and downloaded the newest one using your link above called `libDoxygen_0.4.8-qtc4.12_x64.dll`

I got this message in the `Plugin Loader Message`
```
C:\Users\VM\AppData\Local\QtProject\qtcreator\plugins\4.12.4\libDoxygen_0.4.8-qtc4.12_x64.dll: Cannot load library 
C:\Users\VM\AppData\Local\QtProject\qtcreator\plugins\4.12.4\libDoxygen_0.4.8-qtc4.12_x64.dll: Unknown error 0x000000c1.
```

If I remove this version and try to use another version, example: `Doxygenx4-64.dll`, I got this message:
```
C:\Users\VM\AppData\Local\QtProject\qtcreator\plugins\4.12.4\Doxygen4-x64.dll: Cannot load library 
C:\Users\VM\AppData\Local\QtProject\qtcreator\plugins\4.12.4\Doxygen4-x64.dll: Unknown error 0x000000c1.
```

May be, fixing the Windows plugin version... it can fix the OSX Catalina too....I don't know....

**My Windows machine:**
- Windows 10 x64 (build 1903)
- Qt Creator 4.12.4 - Based on 5.14.2 (MSVC 2017, 32 bit)
- Qt 5.15.0
- MSVC 2019 x64

**Comment by nyckmaia** [comment]
Sorry @francescmm ...it was my mistake!!

I was downloading the `x64` version because my Windows 10 is x64, but my Qt Creator version is `x86`.
So, I replace the plugin to the `Doxygen4-x86.dll` and it works great!

So, Windows ok!

Is there any test that I can try on my OSX Catalina machine to help you to understand the OSX plugin problem?

Thank you!

**Comment by francescmm** [comment]
Happy to see it works, @nyckmaia!

About the OSX Catalina I'm quite lost on it. I need to check the specs I'm using for the CI system and probably connect to it via SSH to see if I'm setting the right path for the DYLD_LIBRARY_PATH variable.

I'll let you know in a while. Thanks for all the comments! :D

**Comment by fpoussin** [comment]
@francescmm Thanks for helping!
Could you submit a pull request so that I integrate the changes here?

**Comment by francescmm** [comment]
Done in #38. I move to Qt Creator 4.13 since it is the latest version.

**Comment by fpoussin** [comment]
Actually the code doesn't need changes, I already had published the binaries for 1.12 but forgot to fix the filenames.
I will add the 1.13 binaries as well.
